util.go

```diff
@@ -63,3 +63,30 @@ func removeTimestampFieldDefinition(fields []string) []string {
 	fields = removeField(fields, "time")
 	return removeField(fields, "sequence_number")
 }
+
+func mapKeyList(m interface{}) []string {
```

Comment:
maybe just toss these at the bottom of `server.go` or `database.go`? I think you might make @benbjohnson cry with something generic like `util`